src/verso/Verso/Doc/TeX.lean

```diff
@@ -26,7 +34,7 @@ public def Options.lift [MonadLiftT m m'] (opts : Options g m) : Options g m' :=
   opts.reinterpret MonadLiftT.monadLift
 
 public abbrev TeXT (genre : Genre) (m : Type → Type) : Type → Type :=
-  ReaderT (Options genre m × genre.TraverseContext × genre.TraverseState) m
+  ReaderT (Options genre m × genre.TraverseContext × genre.TraverseState × TeXContext) m
```

Comment:
Why not nest `ReaderT`here? Then you can use `readThe` to get individual fields.